src/predict.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+import torch
+from stqdm import stqdm
+
+from . import config as cf
+from src.output import write_audio
+
+def load_model():
+    from silero import silero_tts
+
+    model, _ = silero_tts(language=cf.LANGUAGE,
+                                speaker=cf.MODEL_ID)
```

Comment:
You're free to ignore linewidth limits like this. Feel free to take up to screen width per line (say 150 characters per line). The old 70 character limit is kind of old-fashioned now.